extensions/amp-pinterest/amp-pinterest.md

```diff
@@ -95,6 +95,10 @@ Contains the fully-qualified URL of the image intended to be pinned. If the pin
 
 Contains the default description that appears in the pin create form; please choose carefully, since many Pinners will close the form without pinning if it doesn't make sense.
 
+##### alt
```

Comment:
This should go under the **Embedded Pin Widget section** section of the doc.

Reply:
Done